src/SFA.DAS.Roatp.CourseManagement.Web/Controllers/AddProviderContact/ConfirmUpdateStandardsController.cs

```diff
@@ -12,23 +12,28 @@ namespace SFA.DAS.Roatp.CourseManagement.Web.Controllers.AddProviderContact;
 
 public class ConfirmUpdateStandardsController(ISessionService _sessionService) : ControllerBase
 {
-    public const string ViewPath = "~/Views/AddProviderContact/UpdateStandards.cshtml";
+    public const string ViewPathEmailAndPhone = "~/Views/AddProviderContact/UpdateStandardsPhoneAndEmail.cshtml";
+    public const string ViewPathEmail = "~/Views/AddProviderContact/UpdateStandardsEmailOnly.cshtml";
+    public const string ViewPathPhone = "~/Views/AddProviderContact/UpdateStandardsPhoneOnly.cshtml";
+
 
     [HttpGet]
     public IActionResult UpdateStandardsEmailAndPhone(int ukprn)
     {
-        var providerContactModel = _sessionService.Get<ProviderContactSessionModel>();
+        var sessionModel = _sessionService.Get<ProviderContactSessionModel>();
 
-        if (providerContactModel == null) return RedirectToRoute(RouteNames.ReviewYourDetails, new { ukprn = Ukprn });
+        if (sessionModel == null) return RedirectToRoute(RouteNames.ReviewYourDetails, new { ukprn = Ukprn });
 
         var model = new ProviderContactUpdateStandardsViewModel
```

Comment:
It would be easy on eyes if the names are same for views, view models, controller actions.

Reply:
ok I've done some renaming